extra/1-radio-stations.js

```diff
@@ -25,12 +33,20 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
+const getStations=()=>{
+let allFrequencies=getAllFrequencies();
+let availableWaves=[];
+allFrequencies.map(item=>{ isRadioStation(item)?availableWaves=[...availableWaves,item]:null});
```

Comment:
We'll cover the Array map method more in the Week 4 lesson. This is not really the typical way to use Array map.
Normally, we'll use it to map each of the values of an array to another value, and then return the whole array. Here, it looks like you're using it to populate a different array (availableWaves). We can discuss this more if you have some time.